airgun/views/settings.py

```diff
@@ -1,6 +1,8 @@
 from wait_for import wait_for
 from widgetastic.widget import Table
 from widgetastic.widget import Text
+from widgetastic_patternfly import Button
+from widgetastic_patternfly import View
```

Comment:
This way it caused Travis to fail when building docs.
Import from `widgetastic.widget` directly:

```suggestion
from widgetastic.widget import View
```